presentation/src/main/java/me/androidbox/presentation/event/screen/EventScreenState.kt

```diff
@@ -5,12 +5,18 @@ import androidx.compose.runtime.mutableStateListOf
 import me.androidbox.component.agenda.AgendaActionType
 import me.androidbox.component.agenda.VisitorType
 import me.androidbox.component.event.VisitorInfo
+import java.time.ZoneId
+import java.time.ZonedDateTime
 
 data class EventScreenState(
     val listOfPhotoUri: List<Uri> = mutableStateListOf<Uri>(),
     val selectedVisitorType: VisitorType = VisitorType.ALL,
     val selectedVisitor: VisitorInfo? = null,
     val eventTitle: String = "New Event",
-    val eventDescription: String = "New Description",
+    val eventDescription: String = "Description",
     val selectedAgendaActionType: AgendaActionType = AgendaActionType.DELETE_EVENT,
+    val startTimeDuration: ZonedDateTime = ZonedDateTime.now(ZoneId.systemDefault()),
+    val endTimeDuration: ZonedDateTime = ZonedDateTime.now(ZoneId.systemDefault()),
+    val startDateDuration: ZonedDateTime = ZonedDateTime.now(ZoneId.systemDefault()),
+    val endDateDuration: ZonedDateTime = ZonedDateTime.now(ZoneId.systemDefault())
```

Comment:
All these are not really durations, but rather a fixed time/date

Reply:
Updated to startTime, startDate, endTime, and endDate